.github/workflows/provider.yml

```diff
@@ -55,13 +55,13 @@ jobs:
         with:
           path: artifacts
 
-      - run: chmod +x "./artifacts/wasmcloud-aarch64-apple-darwin/bin/${{ inputs.name }}-provider"
-      - run: chmod +x "./artifacts/wasmcloud-aarch64-unknown-linux-musl/bin/${{ inputs.name }}-provider"
-      - run: chmod +x "./artifacts/wasmcloud-x86_64-apple-darwin/bin/${{ inputs.name }}-provider"
-      - run: chmod +x "./artifacts/wasmcloud-x86_64-pc-windows-msvc/bin/${{ inputs.name }}-provider.exe"
-      - run: chmod +x "./artifacts/wasmcloud-x86_64-unknown-linux-musl/bin/${{ inputs.name }}-provider"
+      - run: chmod +x "./artifacts/${{ inputs.name }}-provider-aarch64-apple-darwin/bin/${{ inputs.name }}-provider"
+      - run: chmod +x "./artifacts/${{ inputs.name }}-provider-aarch64-unknown-linux-musl/bin/${{ inputs.name }}-provider"
+      - run: chmod +x "./artifacts/${{ inputs.name }}-provider-x86_64-apple-darwin/bin/${{ inputs.name }}-provider"
+      - run: chmod +x "./artifacts/${{ inputs.name }}-provider-x86_64-pc-windows-gnu/bin/${{ inputs.name }}-provider.exe"
```

Comment:
I will need to double check here, does this have any negative impact for Windows users? E.g. does the `-gnu` target run just fine on a regular Windows 11 machine? It's been quite a while since figuring out the difference between these targets practically.

Reply:
As far as I understand it, this mostly matters if you're integrating in with other windows tools/libraries. Since this is just a self-contained binary, I think this might be the better choice. Also, I don't know if you can cross-compile msvc on linux, but definitely can with gnu

Reply:
`*-gnu` works just fine on Windows and that is tested in CI.
msvc is Windows-only indeed, since it requires Visual Studio and is really only required to link to things built specifically for Visual Studio toolchain on Windows, which we don't do.
See also https://rust-lang.github.io/rustup/installation/windows.html